flink-streaming-java/src/main/java/org/apache/flink/streaming/api/operators/KeyedProcessOperator.java

```diff
@@ -79,7 +79,6 @@ public void onEventTime(InternalTimer<K, VoidNamespace> timer) throws Exception
 
 	@Override
 	public void onProcessingTime(InternalTimer<K, VoidNamespace> timer) throws Exception {
-		collector.setAbsoluteTimestamp(timer.getTimestamp());
```

Comment:
This should call `eraseTimestamp()` because we might still have a timestamp set from processing some previous elements. Same for the other occurrences in the code.

Reply:
@aljoscha Thanks for pointing this out. Fixed!